app/cdap/components/WranglerV2/SearchField/index.tsx

```diff
@@ -0,0 +1,155 @@
+/*
+ * Copyright © 2023 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+import React from 'react';
+import { Box, Divider, Input, InputProps, Link, Typography } from '@material-ui/core';
+import { grey, lightGreen, blue } from '@material-ui/core/colors';
+import CloseOutlinedIcon from '@material-ui/icons/CloseOutlined';
+import InfoOutlinedIcon from '@material-ui/icons/InfoOutlined';
+import T from 'i18n-react';
+import styled from 'styled-components';
+
+export interface IDirectiveSuggestionProps {
```

Comment:
use Props word in interface only if the interface is for the component
plz update across
just use IDirectiveSuggestions, etc.